public/angular/templates/settings.html

```diff
@@ -4,65 +4,57 @@
   </ul>
 </div>
 
-<div ng-controller="FetchingController">
-  <table class="table table-settings">
-    <tbody>
-      <tr>
-        <td class="col-sm-3">
-          <label class="settings-label">Fetching</label>
-        </td>
-        <td>
-          <div data-toggle="buttons">
-            <label ng-click="toggle(false)"
-                   class="toggle-item ng-class:{'active':on===false}">
-              Off
-            </label><label ng-click="toggle(true)"
-                   class="toggle-item ng-class:{'active':on===true}">
-              On
-            </label>
-          </div>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+<div class="row" ng-controller="FetchingController">
+  <div class="col-sm-5">
```

Comment:
What is this view code doing in this PR? Was this left over from another feature, or?